### DISCUSSION TITLE
zh/reference/documents/high_concurrency_implementation_principle

### DISCUSSION THREAD

**Discussion by giscus** [ZLMediaKit/docs#16] [Announcements]
# zh/reference/documents/high_concurrency_implementation_principle

项目介绍 ZLMediaKit (https://github.com/xiongziliang/ZLMediaKit)是一套高性能的流媒体服务框架，目前支持 rtmp/rtsp/hls/http-flv 流媒体协议。该项目已支持 linux、macos、windows、ios、android 平台，支持的编码格式包括 H264、AAC、H265（仅 ...

https://docs.zlmediakit.com/zh/reference/documents/high_concurrency_implementation_principle.html

**Comment by Jerry89722** [comment]
The image is broken.

>图挂了

`TRANS_BY_GPT4`
